### ISSUE TITLE
The size of the Event description TextCtrl is too small. Can the proportion attribute be set to 1?

### ISSUE THREAD

**Issue by hwf1324** [josephsl/evtTracker#3]
As the title suggests, the Event description text box is a bit small.

I have studied the code of NVDA guiHelper for a while, but I cannot find a way to add another control and set the `proportion` or `flag` properties when using `LabeledControlHelper`

**Comment by XLTechie** [comment]
@hwf1324 Here is a [beta version](https://github.com/opensourcesys/evtTracker/releases/download/v24.1.100-beta/evtTracker-24.1.100.nvda-addon) that might solve this problem. It is also 24.1.100 in the Add-on Store beta channel.

I can't tell if it solves it or not, because I can't look at it.

**Please let me know.**

**Comment by hwf1324** [comment]
> @hwf1324 Here is a [beta version](https://github.com/opensourcesys/evtTracker/releases/download/v24.1.100-beta/evtTracker-24.1.100.nvda-addon) that might solve this problem. It is also 24.1.100 in the Add-on Store beta channel.
> 
> I can't tell if it solves it or not, because I can't look at it.
> 
> **Please let me know.**

@XLTechie 

Unfortunately it doesn't solve the problem. I installed version 24.1.100 and the event description text box is still the same size.

The `contentsSizer` is vertical, so the `proportion` property is useless as it only controls the scaling of the `BoxSizer` along the main direction.

I created a PR that solves this problem.

**Comment by hwf1324** [comment]
Another problem is that the current table and description in the vertical proportion of the description of some not high enough, can only display two or three lines of text, if you can set the description of the height of the proportion of higher would be better.
